web/tests/narrow.test.js

```diff
@@ -512,6 +512,17 @@ run_test("show_empty_narrow_message", ({mock_template}) => {
         empty_narrow_html("translated: This user does not exist!"),
     );
 
+    set_filter([
+        ["sender", "alice@example.com"],
+        ["is", "Rome"],
```

Comment:
A stream name wouldn't return anything with the "is" operator.

I'd use one of the valid values for the operator for the test ("private", "starred", "mentioned", "alerted", "unread", "resolved"). 

And maybe add a comment above this cal to  `set_filter` for expected value/case we're testing here.

Reply:
Sure, working on it.